cmd/test.go

```diff
@@ -448,12 +448,14 @@ func (t *Test) GetCmd() *cobra.Command {
 					IgnoreOrdering:     ignoreOrdering,
 					PassthroughHosts:   passThroughHosts,
 				}, enableTele)
-
-				cmd := exec.Command("sudo", "chmod", "-R", "777", path)
-				err = cmd.Run()
-				if err != nil {
-					t.logger.Error("failed to set the permission of keploy directory", zap.Error(err))
-					return err
+				_, err := os.Stat(path)
+				if !os.IsNotExist(err) {
+					cmd := exec.Command("sudo", "chmod", "-R", "777", path)
+					err = cmd.Run()
```

Comment:
instead of directly using this use the util function as this would give uncontrolled security path error.

Reply:
@shivamsouravjha  Done & Tested.